packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -69,7 +70,8 @@ const POISubmission = ({
             switch (identity_last_attempt.service) {
                 case service_code.idv: {
                     if (Number(idv.submissions_left) > 0 || Number(onfido.submissions_left) > 0) {
-                        setSubmissionStatus(submission_status_code.selecting);
+                        if (submission_status !== submission_status_code.complete)
```

Comment:
```suggestion
                        if ((Number(idv.submissions_left) > 0 || Number(onfido.submissions_left) > 0) && submission_status !== submission_status_code.complete)
```
Probably can combine these two statements

Reply:
@yashim-deriv yes that can be done , do you want me update the PR?

Reply:
please do~

Reply:
@yashim-deriv done , please check